int64 command
int64 target
---
bool valid_action


